Gemfile

```diff
@@ -8,6 +8,7 @@ end
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.1.6'
+gem 'bootstrap-sass', '3.3.7'
```

Comment:
1. Xóa toàn bộ comment trong file này
2. Ruby dùng nháy đôi
Em fix toàn bộ file này luôn nhé